.gitignore

```diff
@@ -1,61 +1,23 @@
-# Logs
-logs
-*.log
-npm-debug.log*
-yarn-debug.log*
-yarn-error.log*
-
-# Runtime data
-pids
-*.pid
-*.seed
-*.pid.lock
-
-# Directory for instrumented libs generated by jscoverage/JSCover
-lib-cov
-
-# Coverage directory used by tools like istanbul
-coverage
-
-# nyc test coverage
-.nyc_output
-
-# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
-.grunt
-
-# Bower dependency directory (https://bower.io/)
-bower_components
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
-# node-waf configuration
-.lock-wscript
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
 
-# Compiled binary addons (https://nodejs.org/api/addons.html)
-build/Release
+# testing
+/coverage
 
-# Dependency directories
-node_modules/
-jspm_packages/
+# production
+/build
 
-# TypeScript v1 declaration files
-typings/
+# misc
+.DS_Store
+.env.local
```

Comment:
Will we use .env files? Looks like we need to review this file. In code above, you add `build` folder to ignore list, did TS build use `build` folder by default? I'm not sure, in `tsconfig.json` you are not set `outDir`. But all this ok, we can review all this later.

Reply:
we need .env files here for build. 

but we wouldn't use it for production as only static files are required there

About build dir - yes, it's the default path for CRA